test/addons-napi/common.h

```diff
@@ -0,0 +1,44 @@
+// Empty value so that macros here are able to return NULL or void
+#define NAPI_RETVAL_NOTHING  // Intentionally blank #define
+
+#define GET_AND_THROW_LAST_ERROR(env)                                    \
+  const napi_extended_error_info* error;                                 \
```

Comment:
Should this check if the error is reporting an Exception. IF so we probably want to rethrow that instead of throwing a new  Error.